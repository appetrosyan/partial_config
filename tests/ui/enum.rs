use partial_config::HasPartial;

#[derive(Debug, HasPartial)]
pub enum Conf {
    Hello, World
}

fn main() {}
