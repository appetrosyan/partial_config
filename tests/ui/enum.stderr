error: Enums are not supported.

         = help: While it is possible to support `enum`s in principle, this is most likely an X-Y problem. You should use `partial_config` to build your internal `enum` with an extra layer.

 --> tests/ui/enum.rs:4:10
  |
4 | pub enum Conf {
  |          ^^^^
